packages/verified-fetch/src/utils/request-headers.ts

```diff
@@ -0,0 +1,51 @@
+export function getHeader (headers: HeadersInit | undefined, header: string): string | undefined {
+  if (headers == null) {
+    return undefined
+  }
+  if (headers instanceof Headers) {
+    return headers.get(header) ?? undefined
```

Comment:
The `?? undefined` is redundant?

```suggestion
    return headers.get(header)
```